core/payment-driver/zksync/src/zksync/faucet.rs

```diff
@@ -11,16 +11,16 @@ use tokio::time::delay_for;
 
 // Workspace uses
 use ya_payment_driver::{db::models::Network, model::GenericError};
+use ya_utils_networking::srv_resolver;
 
 // Local uses
 use crate::zksync::wallet::account_balance;
 
-const DEFAULT_FAUCET_ADDR: &str = "http://3.249.139.167:5778/zk/donatex";
```

Comment:
Can SRV record contain not only IP/name + port, but also part of the path  (`/zk/donatex`)

Reply:
I don't know if that's possible. Please verify and adjust the code if not.